CHANGELOG.md

```diff
@@ -1,11 +1,174 @@
 # Changelog
 
-### 1.15.1 - Thursday 22 October
+### 1.16.0 - Wednesday 4 November 2020
+
+- [Session Recording (Beta)](https://github.com/PostHog/posthog/issues/1846)
+
+![Session Recording Page Screenshot](https://posthog.com/static/dec14fdf98d81deada734c03126d482f/2cefc/session-recording.png)
+
+Given that our mission at PostHog is to increase the number of successful projects in the world, session recording felt like a feature that fits in perfectly with that goal.
+
+PostHog already provides various features to help you understand and improve your UX - but watching real users use your product is a _whole other ball game_. 
+
+With PostHog's session recording, you are able to truly feel the pain points of your users first-hand, seeing where they get stuck, debugging exceptions faster, and making your UX smoother. 
+
+![Session Recording Screenshot](https://posthog.com/static/677bd4dc1f4ff4c2b0b2509e66f1e7ea/2cefc/session-recording-ss.png)
+
+Additionally, you can do so while still preserving the privacy of your users, by determining what shouldn't be captured, as well as being able to turn session recording on and off as you wish.
+
+However, please note that our session recording feature is in **Beta** at the moment. This means that it can be unstable and have bugs. To report bugs you find while using it, please [open an issue for us on GitHub](https://github.com/PostHog/posthog/issues). 
+
+
+- [Plugins (Beta)](https://github.com/PostHog/posthog/issues/1896)
+
+![Plugins Screenshot](https://posthog.com/static/f84e34e19a7715f563dabe7c2d3ca823/2cefc/plugins.png)
+
+Plugins is another **Beta** feature that we're extremely excited for. Currently only available for self-hosted instances, plugins allow you to add additional logic to your event processing pipeline, in order to do things like enrich your data or send it somewhere else, like a data warehouse. 
```

Comment:
I'd just replace "like a data warehouse" with "for example a data warehouse", because at first I got the impression that PostHog is the warehouse. 😄